src/qml/pages/onboarding/OnboardingBlockclock.qml

```diff
@@ -20,6 +20,8 @@ InformationPage {
     }
     bold: true
     headerText: qsTr("The block clock")
-    description: qsTr("The Bitcoin network targets a new block every\n10 minutes. Sometimes it's faster and sometimes slower.\n\nThe block clock indicates each block on a dial\nthat represents the current day.")
+    description: qsTr("The Bitcoin network targets a new block every 10 minutes. " +
+        "Sometimes it's faster and sometimes slower. The block clock indicates each " +
+        "block on a dial that represents the current day.")
```

Comment:
The design still specifies that there is atleast one line of text in the onboarding pages text that is on a separate line: https://www.figma.com/file/ek8w3n3upbluw5UL2lGhRx/Bitcoin-Core-App-Design?node-id=5952%3A389674&t=yj4lHUbV7gL4ByjQ-4

Reply:
You can add in a subtext here in the information page control, this is aliased to the subtext of the header control.

Reply:
ill try the subtext out

Reply:
Ill look at subtext, but I actually just removed too many line breaks. Ill update